GTG/gtk/browser/main_window.py

```diff
@@ -260,6 +261,17 @@ def _init_widget_aliases(self):
 
         self.tagpopup = TagContextMenu(self.req, self.app)
 
+        # Setup search syntax popup button
+        self.search_entry.set_icon_from_icon_name(Gtk.EntryIconPosition.SECONDARY, 'help-faq-symbolic')
+        self.search_entry.connect('icon-release', self.on_icon_click)
+
+
+    def on_icon_click(self, entry, icon_pos, event):
```

Comment:
I'd suggest `on_search_box_icon_click` to be less ambiguous.
You could also add typehints here.